drivers/spi/spi_ll_stm32.c

```diff
@@ -131,6 +130,10 @@ static int spi_stm32_dma_tx_load(const struct device *dev, const uint8_t *buf,
 	/* give the fifo mode from the DT */
 	blk_cfg->fifo_mode_control = data->dma_tx.fifo_threshold;
 
+        /* give the circular mode from the DT */
+        blk_cfg->source_reload_en = stream->dma_blk_cfg.source_reload_en;
+        blk_cfg->dest_reload_en = stream->dma_blk_cfg.dest_reload_en;
+
```

Comment:
I realised the step is redundant because of line 102 `blk_cfg = &stream->dma_blk_cfg;`